tarea 2/Luis Ernesto/fecha.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fecha</title>
+</head>
+<body>
+    <script>
+        (function(){
+
+            let fecha = new Date (); 
+
+            let semana = ["Domingo", "Lunes", "Martes", "Miercoles", "Jueves", "Sabado"]; //arreglo para ajustar el dia de la semana
+            let meses = [ "Enero", "Febrero", "Marzo", "Abril", "Mayo", "junio", "julio", "Agosto", "Septiembre", "Octubre", "Noviembre", "Diciembre"]; //arreglo para ajustar el mes de año
```

Comment:
Te sugiero ser consistente con el uso de mayúsculas y minúsculas, además del comentario de `const`.

```suggestion
            const meses = [ "Enero", "Febrero", "Marzo", "Abril", "Mayo", "Junio", "Julio", "Agosto", "Septiembre", "Octubre", "Noviembre", "Diciembre"]; //arreglo para ajustar el mes de año
```